.changesets/breaking_lrlna_rename_metrics_to_use_dots.md

```diff
@@ -0,0 +1,18 @@
+### Update all metrics to use `.` naming convention ([PR #6653](https://github.com/apollographql/router/pull/6653))
+
+Some of the older metrics in the router were not using the most recent `.` naming convention and were instead separated by `_`. For consistency purposes, the following metrics are renamed to use the up-to-date `.` notation:
+
+- `apollo_router_opened_subscriptions` -> `apollo.router.opened.subscriptions`
+- `apollo_router_cache_hit_time` -> `apollo.router.cache.hit.time`
+- `apollo_router_cache_size` -> `apollo.router.cache.size`
+- `apollo_router_cache_hit_count` -> `apollo.router.cache.hit.count`
+- `apollo_router_cache_miss_time` -> `apollo.router.cache.miss.time`
+- `apollo_router_cache_miss_count` -> `apollo.router.cache.miss.count`
+- `apollo_router_state_change_total` -> `apollo.router.state.change.total`
+- `apollo_router_span_lru_size` -> `apollo.router.exporter.span.lru.size`
```

Comment:
Maybe this one should be separated into a separate list for visibility since it also adds `exporter`, while the others just replace the underscore with a dot.

Reply:
how about an additional asterisk with a note about `exporter` like this? 4b931a7b48d7ebaa3ff94a6913ef96c6d8381133